101_Pythonvariablen.ipynb

```diff
@@ -561,39 +637,29 @@
    "outputs": [],
    "source": [
     "x = 10\n",
-    "\"Der Wert von x ist {}\".format(x)"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "Ein Platzhalter wird durch eine öffnende `{` und schließende `}` geschweifte Klammer begrenzt und kann eine Bezeichnung des Platzhalters sowie Formatierungsoptionen beinhalten:"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "\"{raw_value} ist gerundet {rounded_value:.3f}.\".format(raw_value=2/3, rounded_value=2/3)"
+    "y = 3.1415\n",
+    "print(f\"Der Wert von x ist {x} und Pi ist ungefähr {y}\")\n",
+    "\n",
+    "name1 = \"Anna\"\n",
+    "name2 = \"Bob\"\n",
+    "print(f\"Hallo {name1} und {name2}!\")"
```

Comment:
hier sollten wir aufpassen. f-strings sind relativ neu, je nach python version sind die noch nicht unterstützt. wenn die leute älteren code anschauen wird das ältere str.format() häufiger vorkommen.

Reply:
hmm, ja. Also im orignalkurs waren die .format()-sachen drin... Hab's dann geändert, weil f-strings einfacher zu verstehen sind ihmo...
f-strings gibt's schon seit 3.6, also sind relativ alt :wink: :older_adult: 
Keine starke meinung hier. Können uns für eins entscheiden und nen link zu nem Tutorial einfügen, in dem die anderen beschrieben sind... z.B. https://realpython.com/python-string-formatting/

Reply:
jo auch keine starke Meinung hier. Ich bilde mir ein die alten format strings häufiger zu sehen, aber das ist nur nen hunch.